package.json

```diff
@@ -31,7 +31,8 @@
     "flow": "flow status",
     "flow-check": "flow check",
     "coverage": "./scripts/generate-coverage.sh",
-    "build": "./scripts/build.sh"
+    "build": "./scripts/build.sh",
+    "quick-build": "./scripts/quick-build.sh"
```

Comment:
@akmorrow13: just curious about why you need this. `npm run watch` should be fast enough for quick turnarounds while developing, but if that's not useful maybe we should split the `build.sh` as you did.